site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-3.0.x-and-3.11.x-End-of-Life-Announcement.adoc

```diff
@@ -0,0 +1,28 @@
+= Farewell to Apache Cassandra 3.0.x and 3.11.x - End of Life Announcement
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: May 15, 2023
+:page-post-author: The Apache Cassandra Community
+:description: Apache Cassandra 3.0.x and 3.11.x - End of Life Announcement
+:keywords: 
+
+Dear Apache Cassandra Community,
+
+It is with mixed feelings that we announce the end of life (EOL) for Apache Cassandra versions 3.0 and 3.11, effective from the release of version 5.0 in Q4 of 2023. For more than half a decade, these versions have been a vital part of the open-source community, empowering countless users and organizations to build, maintain, and scale distributed databases easily and efficiently. Since the initial release of version 3.0 in November 2015, and the release of 3.11 in June 2017, we have seen enormous growth in the adoption of Apache Cassandra. With your support, these versions have successfully tackled challenges in data management, performance, and scalability. They have been the foundation for numerous applications and businesses around the globe, helping them thrive in today's data-driven world.
+
+However, as we continue to innovate and develop the Apache Cassandra project, we must focus our resources on the latest versions that embody the best of our community's expertise and efforts. As a result, the project committers will no longer maintain the 3.0 and 3.11 branches. We would like to express our heartfelt gratitude to all contributors, committers, and users who have been part of the journey of these remarkable versions.
+
+We understand that many organizations still rely on these versions for their day-to-day operations. The Apache Cassandra community ​will continue to ​evaluate security patches ​contributed to unmaintained branches and will work with​​ and enable contributors who wish to extend these EOLs, should any step forward. CVE fixes may be applied to unmaintained versions as decided on a case-by-case basis. However, the current body of committers feels that they cannot take that task alone and that their efforts are best focused first on moving Cassandra forward, keeping it the scalable database de jure. 
```

Comment:
Pardon my ignorance but is it meant to be "database de jour" (French for "DB of the day")? Doesn't "database de jure" mean "DB by law" in Latin?

```suggestion
We understand that many organizations still rely on these versions for their day-to-day operations. The Apache Cassandra community ​will continue to ​evaluate security patches ​contributed to unmaintained branches and will work with​​ and enable contributors who wish to extend these EOLs, should any step forward. CVE fixes may be applied to unmaintained versions as decided on a case-by-case basis. However, the current body of committers feels that they cannot take that task alone and that their efforts are best focused first on moving Cassandra forward, keeping it the scalable database de jour. 
```

Reply:
de jure is correct